commands/link.js

```diff
@@ -4,31 +4,40 @@ const previous = [];
 
 // We will link all previous with the next one, this might not be needed but we do it
 // because it's easy and it does not really hurt in any way.
-const linkPrevious = (name) => {
-    const links = previous.map((prev) => `npm link ${prev}`);
+const linkPrevious = (name, yarn) => {
+    const pkg = yarn ? 'yarn' : 'npm';
+
+    const links = previous.map((prev) => `${pkg} link ${prev}`);
     previous.push(name);
-    links.push('npm link');
+    if (yarn) {
+        links.push('yarn install');
```

Comment:
This is just nitpicking. It really doesn't matter, but change 'yarn install' to ${pkg} install? Even though if (yarn) is yarn.